applications/FreeSurfaceApplication/custom_utilities/edgebased_levelset_substep.h

```diff
@@ -390,18 +357,16 @@ namespace Kratos
                 const double havg_i = mHavg[i_node];
                 const double hmin_i = mHmin[i_node];
                 const double eps_i = mEps[i_node];
-                // const double d_i = mD[i_node];
                 double nu = mViscosity[i_node];
-                //            const double lindarcy_i = mA[i_node];
-                //            const double nonlindarcy_i = mB[i_node];
                 double vel_norm = norm_2(v_i);
-                // double porosity_coefficient = ComputePorosityCoefficient(nu, vel_norm, eps_i, d_i);
-                //            double porosity_coefficient = ComputePorosityCoefficient( vel_norm, eps_i, lindarcy_i, nonlindarcy_i);
+
                 vel_norm /= eps_i;
+
                 // use CFL condition to compute time step size
-                double delta_t_i = 1.0 / (vel_norm / hmin_i + nu / (hmin_i * hmin_i) /*+ porosity_coefficient*/);
-                double delta_t_i_avg = 1.0 / (vel_norm / havg_i + nu / (havg_i * havg_i) /*+ porosity_coefficient*/);
+                double delta_t_i = 1.0 / (vel_norm / hmin_i + nu / (hmin_i * hmin_i));
+                double delta_t_i_avg = 1.0 / (vel_norm / havg_i + nu / (havg_i * havg_i));
```

Comment:
Not sure at all, but I've the feeling that you can make these `const`.

Reply:
Only delta_t_i_avg can be const